WordPress/src/main/java/org/wordpress/android/widgets/WPNetworkImageView.java

```diff
@@ -183,6 +187,13 @@ private void loadImageIfNecessary(final boolean isInLayoutPass) {
             }
         }
 
+        // skip this URL if a previous request for it returned a 404
+        if (mUrlSkipList.contains(mUrl)) {
```

Comment:
> I wonder If we should call add and contains in a synchronized way.

I'm pretty sure `mUrlSkipList` is only accessed from the main thread, so synchronization isn't necessary.